language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala

```diff
@@ -90,7 +91,15 @@ object TestUtil {
     Map[String, Filter](partyName -> NoFilter.instance).asJava
   )
 
-  def sendCmd(channel: Channel, partyName: String, cmds: Command*): Empty = {
+  def sendCmd(channel: Channel, partyName: String, update: Update[_]): Empty = {
+    sendCmd(channel, partyName, List(update))
+  }
+
+  def sendCmd(channel: Channel, partyName: String, updates: List[Update[_]]): Empty = {
+    sendCmd(channel, partyName, updates.flatMap(_.commands().asScala): _*)
+  }
+
+  private def sendCmd(channel: Channel, partyName: String, cmds: Command*): Empty = {
```

Comment:
This can just go back to the original code, with `HasCommands` instead of `Command`.